src/components/resourceLinks/ResourceLinks.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import "./ResourceLinks.css"
+const ResourceLinks = ({icon, icon_alt, target, url, icon_only, name, classname}) => {
+    return (
+        <div>
```

Comment:
Deze `div` is niet nodig, want je geeft alleen een `<a>` terug die alles omwikkeld!